src/include/mpir_op.h

```diff
@@ -113,10 +113,10 @@ extern MPIR_Op MPIR_Op_builtin[MPIR_OP_N_BUILTIN];
 extern MPIR_Op MPIR_Op_direct[];
 extern MPIR_Object_alloc_t MPIR_Op_mem;
 
-#define MPIR_Op_add_ref(_op) \
-    do { MPIR_Object_add_ref(_op); } while (0)
-#define MPIR_Op_release_ref(_op, _inuse) \
-    do { MPIR_Object_release_ref(_op, _inuse); } while (0)
+#define MPIR_Op_add_ref(op_p_) \
+    do { MPIR_Object_add_ref(op_p_); } while (0)
+#define MPIR_Op_ptr_release_ref(op_p_, inuse_) \
```

Comment:
This changes a macro name as well as variable name. Both are inconsistent with other object ref macros, such as `MPIR_Comm_`, `MPIR_Request_`. I would prefer to drop this for now, or do a more broad cleanup than just the Op and Datatype macros in the future.

Reply:
This commit is only changing variable name. I have applied the same change to `MPIR_Comm` and `MPIR_Request` structures.

Reply:
This still changes `MPIR_Op_release_ref` -> `MPIR_Op_ptr_release_ref`.

Reply:
Oops. Fixed it now.